test/integration/etcd/data.go

```diff
@@ -191,10 +191,18 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		},
 		// --
 
+		// k8s.io/kubernetes/pkg/apis/discovery/v1
+		gvr("discovery.k8s.io", "v1", "endpointslices"): {
+			Stub:             `{"metadata": {"name": "slicev1"}, "addressType": "IPv4", "protocol": "TCP", "ports": [], "endpoints": []}`,
+			ExpectedEtcdPath: "/registry/endpointslices/" + namespace + "/slicev1",
+		},
+		// --
+
 		// k8s.io/kubernetes/pkg/apis/discovery/v1beta1
 		gvr("discovery.k8s.io", "v1beta1", "endpointslices"): {
 			Stub:             `{"metadata": {"name": "slicev1beta1"}, "addressType": "IPv4", "protocol": "TCP", "ports": [], "endpoints": []}`,
 			ExpectedEtcdPath: "/registry/endpointslices/" + namespace + "/slicev1beta1",
+			ExpectedGVK:      gvkP("discovery.k8s.io", "v1", "EndpointSlice"),
```

Comment:
storage version should remain v1beta1 until 1.22 (see comment above in install about preferring v1beta1 until then)

Reply:
Updated to expect v1beta1 for v1.